### ISSUE TITLE
[Feature request] Autoboot a title/game through a button combination

### ISSUE THREAD

**Issue by iCRON862** [LumaTeam/Luma3DS#1845]
If autoboot the hbmenu was possible, can you make a feature to autoboot a title/game through button combinations specified in config.ini as example? Or can i modify the config.ini to make luma3ds autoboot the title without the hbmenu take over? Thx

**Comment by CyberYoshi64** [comment]
It feels weird that the autoboot option for 3DS uses the same value Luma determines to override with the HBL. I hope this effect isn't intended and wish that the autoboot value is seperated so you can boot other things besides HBL.

**Comment by deco1212** [comment]
That is avaliable through Fastboot3DS

**Comment by mariomadproductions** [comment]
> That is avaliable through Fastboot3DS

How?

**Comment by urherenow** [comment]
Now that’s a lazy question. It’s in the readme (home page) of the fastboot3ds github. 

https://github.com/derrekr/fastboot3DS

**Comment by mariomadproductions** [comment]
> Now that’s a lazy question. It’s in the readme (home page) of the fastboot3ds github.
> 
> https://github.com/derrekr/fastboot3DS

I can't see anything relevant in the readme, or exploring the menu myself.

**Comment by urherenow** [comment]
please learn to read. And don't start discussing other projects in the wrong github.

"You may also want to set up the other boot slots and assign key combos to them. Keep in mind you need one autoboot slot (= a slot with no key combo assigned). If you want to access the fastboot3DS menu at a later point in time, hold the HOME button when powering on the console."

Things just above and continuing in that paragraph, tell you how.

**Comment by mariomadproductions** [comment]
> please learn to read. And don't start discussing other projects in the wrong github.
> 
> "You may also want to set up the other boot slots and assign key combos to them. Keep in mind you need one autoboot slot (= a slot with no key combo assigned). If you want to access the fastboot3DS menu at a later point in time, hold the HOME button when powering on the console."
> 
> Things just above and continuing in that paragraph, tell you how.

This issue is about loading titles, not firms.

**Comment by urherenow** [comment]
You're right. My bad. But it's the most rediculous thing I ever heard. Do you understand what a firm does? It sets up all of the hardware. You can't even boot the home menu without a firm, so the answer is an absolutely eye-rolling NO. You CANNOT directly boot a title without booting a firm.

https://www.3dbrew.org/wiki/FIRM

Now, you might be able to make a firm that loads something other than the home screen, but it's still a firm, and if it worked, Luma or fastboot could boot it. That's a separate project, not a feature to be added. It already boots firms.

**Comment by mariomadproductions** [comment]
I already know all that. Anyway, I understand what you are saying. Sorry to everyone for clogging up the thread.
